app/controllers/api/v1/rents_controller.rb

```diff
@@ -21,6 +21,10 @@ def create
       def rent_params
         params.require(:rent).permit(:user_id, :book_id, :start, :end)
       end
+
+      def set_locale
+        I18n.locale = current_user.try(:locale) || I18n.default_locale
```

Comment:
Since ruby 2.3.0 there is safe navigation. You can do:
```
current_user&.locale || I18n.default_locale
```

Reply:
oka